nixos/modules/services/web-apps/openwebrx.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.openwebrx;
+in
+{
+  options.services.openwebrx = with lib; {
+    enable = mkEnableOption "Enable OpenWebRX Web interface for Software-Defined Radios";
```

```suggestion
    enable = mkEnableOption "OpenWebRX Web interface for Software-Defined Radios";
```

The "Enable" is already supplied by `mkEnableOption`.